project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,354 @@
+            - Misc Extra Refrigerator|EF 19.9
+        costs:
+          - value: 2003.0 # price1: $1699-1799 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in BestBuy
+                          # price2: $8144-8599 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in Home Depot. This price is too high. Therefore, will not refer these prices. 
+                          # 2003.0=1749*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1749=(1699+1799)/2
+                         # Did not find ENERGY STAR most efficient refrigerator or refrigerator with EF 21.9 for 20.9 Cu.Ft. Refrigerator. Therefore, use the same price for EnergryStar and ENERGY STAR most efficient refrigerator.
+                       # Another option: $1400. $1000 for EF 21.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+      - &energystar_most_eff_clothes_washer
+        option: Clothes Washer|EnergyStar Most Efficient
+        apply_logic:
+        - or:
+            - Clothes Washer|Standard
+            - Clothes Washer|EnergyStar
+            - Clothes Washer|EnergyStar, Cold Only
+            - Clothes Washer|EnergyStar More Efficient
+        costs:
+          - value: 1537.0  #price: $ 628-$1999 for ENERGY STAR most efficient front load clothes washer (>2.5 Cu.Ft) in EnergyStar product database
+                         # 1537.0=1313.5*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1313.5=(628+1999)/2
+                           # Another option: $1680. $1200 for replacing clothes washer to MEF2.88 in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 14 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=293&actionId=1068&bcId=4538
+
+      - &energystar_most_eff_dishwasher 
+        option: Dishwasher|240 Rated kWh
+        apply_logic:
+          - or:
+            - Dishwasher|318 Rated kWh
+            - Dishwasher|290 Rated kWh
+            - Dishwasher|270 Rated kWh
+            - Dishwasher|255 Rated kWh
+        costs:
+          - value: 1762.8 #price1: $699.99-2499 for ENERGY STAR Certified diswasher (240 Rated kWh) in BestBuy
+                         #price2: $549.99-1299 for ENERGY STAR Certified diswasher (240 Rated kWh) in Home Depot
+                         # 1762.8=1524.5*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor.
+            multiplier: Fixed (1)
+        lifetime: 11 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=291&actionId=1065&bcId=4484
+                     # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_most_eff_clothes_dryer_electric 
+        option: Clothes Dryer|Electric, Premium, EnergyStar Most Efficient
```

Comment:
Per the notes on the taskboard (and my memory), we were also going to increase the usage multiplier when going to an EnergyStar Most Efficient dryer (because of the smaller volume).

Reply:
I added these multipliers and pushed to the remote.

Reply:
Need to double-check if they should apply to electric, gas, or both, and how the volume variable is used.